recipe/meta.yaml

```diff
@@ -1,22 +1,20 @@
-{% set name = "gts" %}
-{% set version = "0.7.6" %}
-
 package:
-  name: {{ name }}
-  version: {{ version }}
+  name: gts
+  version: 0.7.6
 
 source:
-  - url: http://downloads.sourceforge.net/gts/gts-{{ version }}.tar.gz
+  - url: http://iweb.dl.sourceforge.net/project/gts/gts/0.7.6/gts-0.7.6.tar.gz
     sha256: 059c3e13e3e3b796d775ec9f96abdce8f2b3b5144df8514eda0cc12e13e8b81e
     patches:
       - config.in.win32.patch
       - 0001-Disable-USE_PREDICATES_INIT.patch  # [build_platform != target_platform]
   - path: CMakeLists.txt
 
 build:
-  number: 2
+  number: 3
+  skip: True    # [win32]
   run_exports:
-    - {{ pin_subpackage(name, max_pin='x.x') }}
+    - {{ pin_subpackage('gts', max_pin='x.x') }}
```

Comment:
Was there a reason this had to be made explicit?

Reply:
Quote from the Zen of Python: `Explicit is better than implicit.`

Reply:
fair enough